src/test/kotlin/domain/GameTest.kt

```diff
@@ -192,7 +191,7 @@ class GameTest : DescribeSpec({
                         ),
                     ),
                 )
-            val fixedParticipants = Participants(listOf(Player("pobi")))
+            val fixedParticipants = listOf(Player("pobi", 1000))
```

Comment:
```suggestion
            val fixedParticipants = listOf(Player("pobi", 1_000))
```
요런 식으로 `_` 표기를 통해 자리수를 드러내줄 수도 있습니다 😉